library/src/main/java/com/novoda/viewpageradapter/ViewPagerAdapter.java

```diff
@@ -51,7 +56,7 @@ public V instantiateItem(ViewGroup container, int position) {
      * @param position  the position of the data set that is to be represented by this view
      * @param viewState the state of the view
      */
-    protected void bindView(V view, int position, @Nullable SparseArray<Parcelable> viewState) {
+    private void bindView(V view, int position, @Nullable SparseArray<Parcelable> viewState) {
```

Comment:
:x: please check with internal teams on this - this is probably used. particularly @ouchadam @andrei or the third person on that project who I can't remember :joy: